examples/Dojo/index.html

```diff
@@ -87,49 +89,16 @@ <h2>Checkout Input Form</h2>
     <script>
       dojoConfig = {
         baseUrl: '.',
-        paths: {
-          js: "js"
-        },
-        packages: [
```

Comment:
What was the reason for removing these packages from the config?

Reply:
Doesn't do anything given the other comments, but for now I've left them in.